Source/WebCore/page/ResizeObservation.cpp

```diff
@@ -52,24 +52,26 @@ void ResizeObservation::updateObservationSize(const BoxSizes& boxSizes)
     m_lastObservationSizes = boxSizes;
 }
 
-auto ResizeObservation::computeObservedSizes() const -> BoxSizes
+std::optional<ResizeObservation::BoxSizes> ResizeObservation::computeObservedSizes() const
 {
     if (m_target->isSVGElement()) {
         if (auto svgRect = downcast<SVGElement>(*m_target).getBoundingBox()) {
             auto size = LayoutSize(svgRect->width(), svgRect->height());
-            return { size, size, size };
+            return BoxSizes { size, size, size };
         }
     }
     auto* box = m_target->renderBox();
     if (box) {
-        return {
+        if (box->isSkippedContent())
+            return std::nullopt;
+        return BoxSizes {
             adjustLayoutSizeForAbsoluteZoom(box->contentSize(), *box),
             adjustLayoutSizeForAbsoluteZoom(box->contentLogicalSize(), *box),
             adjustLayoutSizeForAbsoluteZoom(box->borderBoxLogicalSize(), *box)
         };
     }
 
-    return { };
+    return BoxSizes { };
```

Comment:
Probably could even use the two sets of braces here, but probably not better!

Reply:
Kept as-is.